src/azure/managementUrlHelper.ts

```diff
@@ -0,0 +1,53 @@
+import * as core from '@actions/core';
+import * as exec from '@actions/exec';
+import * as io from "@actions/io";
+
+export class ManagementUrlHelper{
+
+    public static async getBaseUrl(): Promise<string> {
+        if(!this._baseUrl) {            
+            let azCloudDetails = JSON.parse(await ManagementUrlHelper.executeAzCliCommand('cloud show'));
```

Comment:
Should we wrap the baseUrl derivation logic in `try..catch`, just to be safe?

Reply:
try catch added.